receiver/chronyreceiver/README.md

```diff
@@ -0,0 +1,61 @@
+# Chrony Receiver
+
+| Status                   |           |
+| ------------------------ | --------- |
+| Stability                | [alpha]   |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
+
+The _Chrony Receiver_ is a pure go implementation of the command `chronyc tracking` to allow for
+portability across systems and platforms. All of the data that would typically be captured by
+the tracking command is made available in this receiver, see [documentation](./documentation.md) for
+more details.
+
+## Configuration
+
+### Default
+
+By default, the `chrony` recevier will default to the following configuration:
+
+```yaml
+chrony/defaults:
+  address: unix:///var/run/chrony/chronyd.sock # The default port by chronyd to allow cmd access
+  timeout: 10s # Allowing at least 10s for chronyd to respond before giving up
+
+chrony:
+  # This will result in the same configuration as above
+```
+
+### Customised
+
+The following things can be customised:
+
+- address (required) - the address on where to communicate to `chronyd`
+  - The allowed formats are the following
+    - udp://hostname:port
+    - unix:///path/to/chrony.sock (Please note the triple slash)
+  - chronyd can be configured to allow tracking comamands either by a udp port or via the running chronyd unix socket
```

Comment:
Not quite sure I understand the purpose of this comment, especially at this location.

Reply:
I think this was me brain dumping in a covid infused state, I have removed it since it didn't makes sense when rereading.